fusillade/directory/resource.py

```diff
@@ -339,6 +339,11 @@ def create_id(self, name: str, owner: str = None, **kwargs) -> 'ResourceId':
     def get_id(self, *args, **kwargs) -> 'ResourceId':
         return ResourceId(self.name, *args, **kwargs)
 
+    def get_info(self) -> Dict[str, Any]:
+        info = super(ResourceType, self).get_info()
+        info[f'{self.object_type}_type'] = info.pop(f'{self.object_type}_id')
```

Comment:
Just a nit but I think the naming (object type type = object type id) could be clearer?